.goreleaser.yml

```diff
@@ -0,0 +1,26 @@
+before:
+  hooks:
+    - go mod download
+    - go generate ./...
+builds:
+- env:
+  - CGO_ENABLED=0
+- goos:
+  - darwin
+  - linux
+  - windows
+- goarch:
+  - amd64
+archives:
+- replacements:
+    amd64: x86_64
+checksum:
+  name_template: 'checksums.txt'
+snapshot:
+  name_template: "{{ .Tag }}-next"
+changelog:
+  sort: asc
+  filters:
+    exclude:
+    - '^docs:'
+    - '^test:'
```

Comment:
This makes an assumption about the commit message format that is not enforced.  I'd be fine bringing this in but we should put the commit checks in place.

Reply:
This has been removed. All commits will be added to the change log